resources/views/components/tag.blade.php

```diff
@@ -0,0 +1,11 @@
+@props([
+    'label',
+    'link'
+    ])
+<a href="{{$link}}" class="transition ease-in-out duration-300 transition hover:scale-105">
```

Comment:
```suggestion
<a href="{{ $link }}" class="transition hover:scale-105">
```

`ease-in-out duration-300` is not needed as `transition` sets these values for you!

Reply:
there's also no need for the scale effect on this component as it has the background change on hover